inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -148,6 +166,35 @@ void MKLDNNConvolutionNode::getSupportedDescriptors() {
         }
     }
 
+    if (isWinograd()) {
```

Comment:
Please put the comment why we need this WA.

Reply:
done